scripts/lossPersistence/pp.R

```diff
@@ -0,0 +1,273 @@
+# Instalar librerias necesarias
+packagesPrev<- installed.packages()[,"Package"]
+packagesNeed<- list("magrittr", "terra", "raster", "sf", "fasterize", "pbapply", "gdalUtilities")

```

Comment:
Packages fasterize and pbapply are not pre-installed. Please check if they are still necessary, and if yes add them to runners/r-dockerfile.